### ISSUE TITLE
Resource encryption add

### ISSUE THREAD

**Issue by MrSayko** [sunnamed434/BitMono#84]
### Summary

Resource encryption add

### Basic Example

Resource encryption add

### Drawbacks

Resource encryption add

**Comment by sunnamed434** [comment]
Good idea, I'll think about that in the coming week deeply, to understand how to do that, etc, also now I'm thinking about changing the issue templates, this is a bit bothering, at least for me

**Comment by MrSayko** [comment]
> Herhangi bir kaynak şifreleme örneğiniz var mı? Nasıl çalışmalı?

https://github.com/yck1509/ConfuserEx/tree/master/Confuser.Protections/Resources
https://github.com/yck1509/ConfuserEx/blob/master/Confuser.Runtime/Resource.cs

**Comment by sunnamed434** [comment]
> https://github.com/yck1509/ConfuserEx/tree/master/Confuser.Protections/Resources https://github.com/yck1509/ConfuserEx/blob/master/Confuser.Runtime/Resource.cs

Cool, I'll check

**Comment by sunnamed434** [comment]
So, I did some research and as I see in Confuserex exist such issue that also need support for Costura.Fody in case of adding compressor

**Comment by MrSayko** [comment]
No need for Costura.Fody, if you give me discord I can give an example

**Comment by sunnamed434** [comment]
> No need for Costura.Fody, if you give me discord I can give an example

Here's my discord: `sunnamed#9250`
